tests/backends/postgresql/tests.py

```diff
@@ -145,3 +147,29 @@ def test_correct_extraction_psycopg2_version(self):
             self.assertEqual(psycopg2_version(), (4, 2, 1))
         with mock.patch('psycopg2.__version__', '4.2b0.dev1 (dt dec pq3 ext lo64)'):
             self.assertEqual(psycopg2_version(), (4, 2))
+
+
+@unittest.skipUnless(connection.vendor == 'postgresql', 'PostgreSQL tests')
+class CallProcTest(BaseCallProcTest, unittest.TestCase):
+    procedure_without_params_sql = '''
```

Comment:
I feel like double quotes are more often used in triple quoted strings but maybe that's only in docstrings?